addons/account/models/account.py

```diff
@@ -462,6 +462,23 @@ def action_open_reconcile(self):
             'context': action_context,
         }
 
+    def action_open_account(self):
```

Comment:
```suggestion
    def action_view_account(self):
```

Instead of full return can't we read action and use it